### PR TITLE
PPNA-6403 Update documentation based on date_from/date_to API specifi…

### PR DESCRIPTION
**Pull request by janusz-piechota-piwik**
…cation
obraz
obraz
obraz
obraz
obraz
:/

### CODE REVIEW

File: custom_reports/http_api/definitions/query_request.yaml [hunk 1]
```diff
@@ -28,14 +28,22 @@ properties:
     example: [{"column_id":"device_type"},{"column_id":"revenue","transformation_id":"sum"}]
   date_from:
     type: string
-    format: date
-    description: Start date for the query (inclusive). Cannot be used with relative_date field at
-      the same time. Mandatory if ``relative_date`` is not used.
+    oneOf:
+      - format: date
+        description: Start date for the query (inclusive).
```
Comment (mrfijas) [inline custom_reports/http_api/definitions/query_request.yaml:33]:
```suggestion
        description: Start date for the query (inclusive, in website's timezone).
```
dates are also in the timezone, and clients often ask about it

File: custom_reports/http_api/definitions/query_request.yaml [hunk 2]
```diff
@@ -28,14 +28,22 @@ properties:
...
+      - format: datetime
+        description: Start datetime for the query (inclusive, in website's timezone).
```
Comment (mrfijas) [inline custom_reports/http_api/definitions/query_request.yaml:35]: if we use "datetime" here, it's worth unifying it in "Start date (with time) -> Start datetime" also. alternatively we could consult the copy with Joanna

Comment (kororokke) [reply custom_reports/http_api/definitions/query_request.yaml:35]: and also i would use ``` `relative_date` ``` where possible, in one place there is ``` `relative_date` ``` in both places, in some cases only in one place.

Comment (kororokke) [reply custom_reports/http_api/definitions/query_request.yaml:35]: Cannot be used with ``` ``relative_date`` ``` field at the same time.
      Mandatory if ``` ``relative_date`` ``` is not used.

vs

Cannot be used with relative_date field at the same time.
      Mandatory if ``` ``relative_date`` ``` is not used.

File: custom_reports/http_api/definitions/raw_data_real_time_request.yaml [hunk 3]
```diff
@@ -28,11 +28,11 @@ properties:
   date_from:
     type: string
     format: datetime
-    description: Start date (with time) for the query (inclusive, in website's timezone).
+    description: Start datetime for the query (inclusive, in website's timezone).
   date_to:
     type: string
     format: datetime
-    description: End date (with time) for the query (exclusive, in website's timezone).
+    description: End datetime for the query (exclusive, in website's timezone).
```
Comment (janusz-piechota-piwik) [inline custom_reports/http_api/definitions/raw_data_real_time_request.yaml:35]: @mrfijas is it now OK ?

**Review by mrfijas — APPROVED** [review]

**Review by pulina — APPROVED** [review]

**Review by kororokke — APPROVED** [review]

**Pull request merged**